src/test/scala/com/microsoft/hyperspace/goldstandard/PlanStabilitySuite.scala

```diff
@@ -0,0 +1,289 @@
+// scalastyle:off
+
```

Comment:
To Reviewers: This file is build after modifying spark's https://github.com/apache/spark/blob/branch-3.1/sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala